.github/actions/is-release-week-v1/README.md

```diff
@@ -0,0 +1,23 @@
+# is-release-week-v1
+
+A GitHub Action to determine if release should continue to run or stop.
+
+## Inputs
+
+| Name      | Required | Description                                                                            | Default |
+| --------- | -------- | -------------------------------------------------------------------------------------- | ------- |
+| `oddWeek` | yes      | `true` if the action should run on odd weeks, `false` if the action should run on even | `false` |
+
+## Outputs
+
+| Name            | Description                                            | Default |
+| --------------- | ------------------------------------------------------ | ------- |
+| `isReleaseWeek` | `true` if the week of year matches the `oddWeek` input | `false` |
```

Comment:
Saw Steve comment this, I missed it outputs have no defaults its based from the input so this can be - 
```suggestion
| Name            | Description                                           
| --------------- | ------------------------------------------------------ |
| `isReleaseWeek` | `true` if the week of year matches the `oddWeek` input |
```